<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="11" passed="5" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-03T15:26:52 IST" name="Default suite" finished-at="2024-09-03T15:27:53 IST" duration-ms="60829">
    <groups>
    </groups>
    <test started-at="2024-09-03T15:26:52 IST" name="Default test" finished-at="2024-09-03T15:27:53 IST" duration-ms="60829">
      <class name="testCases.LoginPage_Test_Second_Day3">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:26:52 IST" name="setup" finished-at="2024-09-03T15:27:02 IST" duration-ms="9342" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:02 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:02 IST" duration-ms="712" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:02 IST" name="close" finished-at="2024-09-03T15:27:06 IST" duration-ms="3067" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=SUCCESS method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:06 IST" name="setup" finished-at="2024-09-03T15:27:10 IST" duration-ms="4050" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:10 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:10 IST" duration-ms="659" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.LoginPage_Test_Second_Day3.verifyLoginTest(LoginPage_Test_Second_Day3.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:10 IST" name="close" finished-at="2024-09-03T15:27:11 IST" duration-ms="1194" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=FAILURE method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:11 IST" name="setup" finished-at="2024-09-03T15:27:16 IST" duration-ms="4110" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:16 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:16 IST" duration-ms="764" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:16 IST" name="close" finished-at="2024-09-03T15:27:17 IST" duration-ms="1145" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=SUCCESS method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:17 IST" name="setup" finished-at="2024-09-03T15:27:22 IST" duration-ms="4076" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:22 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:27 IST" duration-ms="5757" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:27 IST" name="close" finished-at="2024-09-03T15:27:28 IST" duration-ms="944" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=SUCCESS method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:28 IST" name="setup" finished-at="2024-09-03T15:27:34 IST" duration-ms="5572" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:34 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:34 IST" duration-ms="641" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:34 IST" name="close" finished-at="2024-09-03T15:27:36 IST" duration-ms="1097" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=SUCCESS method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:36 IST" name="setup" finished-at="2024-09-03T15:27:40 IST" duration-ms="4060" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:40 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:40 IST" duration-ms="722" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:40 IST" name="close" finished-at="2024-09-03T15:27:41 IST" duration-ms="1092" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=SUCCESS method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:41 IST" name="setup" finished-at="2024-09-03T15:27:45 IST" duration-ms="4086" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:45 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:46 IST" duration-ms="649" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[namrata_pawar]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_password]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.LoginPage_Test_Second_Day3.verifyLoginTest(LoginPage_Test_Second_Day3.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:46 IST" name="close" finished-at="2024-09-03T15:27:47 IST" duration-ms="1138" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=FAILURE method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:47 IST" name="setup" finished-at="2024-09-03T15:27:51 IST" duration-ms="4056" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:51 IST" name="verifyLoginTest" data-provider="validation" finished-at="2024-09-03T15:27:52 IST" duration-ms="686" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[namrata_narve]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password_sauce]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.LoginPage_Test_Second_Day3.verifyLoginTest(LoginPage_Test_Second_Day3.java:77)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method is-config="true" signature="close(org.testng.ITestResult)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff]" started-at="2024-09-03T15:27:52 IST" name="close" finished-at="2024-09-03T15:27:53 IST" duration-ms="1128" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=FAILURE method=LoginPage_Test_Second_Day3.verifyLoginTest(java.lang.String,java.lang.String)[pri:0, instance:testCases.LoginPage_Test_Second_Day3@793f29ff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- testCases.LoginPage_Test_Second_Day3 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
